### DISCUSSION TITLE
hologram command !rightclick!cmi sell hand

### DISCUSSION THREAD

**Discussion by BlueSteelAUS** [Zrips/CMI#1470] [Enhancement]
can't get command
!rightclick!cmi sell hand
to work on hologram commands
(yes interactive is active)

**Cmi Version (using`/cmi version`):** 
8.4.0.7
**Server Type (Spigot/Paperspigot/etc):** 
paper(527) 1.13.2-R0.1-snapshot

**Comment by LogGits** [comment]
Ok so this is running the command as the console. There could be a way to achieve this but for now i will mark it as a feature request for:
`asConsole!` and `asPlayer!` to work with hologram commands.

**Comment by BlueSteelAUS** [comment]
tyvm.. yeah it's with any command that doesn't require a playerid in the command.. that default to current player.

I spent like 2 days trying to get it to work either as a hologram or on a sign I even tried as an interactive command. none of them will do that or similar commands
